src/locale/languages/en.js

```diff
@@ -157,6 +157,9 @@ export const en = {
     followingList: {
       title: 'Following',
     },
+    followYou: {
+      title: 'Follows you',
+    },
```

Comment:
Can we make it fall back on the English translation if a phrase doesn't exist in a certain language? That would make everything a lot smoother IMO, and make it so we don't have to update every language in small PR's.